.github/workflows/integration-tests.yaml

```diff
@@ -93,7 +93,9 @@ jobs:
           OCTODISABLEOCTOCONTAINERLOGGING: true
           OCTOTESTSKIPINIT: true
           GOMAXPROCS: 1
-          OCTOTESTVERSION: 2023.4
+          OCTOTESTVERSION: latest
+          OCTOTESTIMAGEURL: docker.packages.octopushq.com/octopusdeploy/octopusdeploy
+          DOCKER_AUTH_CONFIG: '{"auths":{"docker:packages:octopushq:com":{"username":"${{ secrets.OCTOPUSHQ_DOCKER_REGISTRY_USER }}","password":"${{ secrets.OCTOPUSHQ_DOCKER_REGISTRY_PASSWORD }}"}}}'
```

Comment:
Not sure if the secrets will actually be expanded properly here. Will need to get access to modify secrets first before I can try it out.

Reply:
For future reference, the variables do get expanded in that expression. However, the Docker auth env var is not needed for the GA workflow agent to authenticate with the Docker registry - hence this has been removed.